app/components/chart-published.tsx

```diff
@@ -121,11 +127,22 @@ export const ChartPublished = (props: ChartPublishedProps) => {
     [configKey, dataSource, metadataPanelStore, state]
   );
 
+  const getPadding = () => {
+    if (isDesktop) return 32;
+    if (isTablet) return 24;
+    if (isMobile) return "24px 16px 8px 16px";
+  };
+
   return (
     <MetadataPanelStoreContext.Provider value={metadataPanelStore}>
       <InteractiveFiltersProvider chartConfigs={state.chartConfigs}>
         {state.layout.type === "dashboard" ? (
-          <>
+          <Box
+            style={{
+              padding: getPadding(),
+              backgroundColor: theme.palette.grey[200],
+            }}
+          >
```

Comment:
I think it would make sense to either use:

```ts
<Box sx={{ p: getPadding(), backgroundColor: "grey.200" }}>...</Box>
```

or

```ts
<div style={{ padding: getPadding(), backgroundColor: theme.palette.grey[200] }}>...</div>
```

or even extract styles to `makeStyles` to decouple the styles definitions from the component body altogether 👀 

The benefit of using `Box` over `div` is that you can use the [sx](https://mui.com/system/getting-started/the-sx-prop/) prop that doesn't require importing `theme` upfront – as you already have it imported for the `isMobile`, ... hooks, you could fall back to a `div` here 👍